Specs/DateTime/Dutch/DateTimeModel.json

```diff
@@ -9974,5 +9974,55 @@
         }
       }
     ]
+  },
+  {
+    "Input": "Ik heb op de vorige koningsdag veel leuke activiteiten gedaan!",
+    "Context": {
+      "ReferenceDateTime": "2020-08-17T00:00:00"
+    },
+    "Comment": "Last Kingsday should be 2020, not 2019. Relative date should first look for a date in the same year.",
```

Comment:
It looks like the implementation of #2236 is still a bit buggy. I ported that PR to Dutch DateTime, however the issue is when a date did happen already in a year.

For example, last Easter has been `2020-04-12`, however using the new relative Holiday feature it will just shift one year. It doesn't have a look if the Holiday did already happen this year.

Is this a correct implementation for English? For Dutch, saying 'last easter' would mean the last one, thus `2020-04-12`.

```
{
  "Text": "last easter",
  "Start": 0,
  "End": 10,
  "TypeName": "datetimeV2.date",
  "Resolution": {
    "values": [
      {
        "timex": "2019-04-21",
        "type": "date",
        "value": "2019-04-21"
      }
    ]
  }
}
```

Using previous will have a null resolution.
```
{
  "Text": "previous easter",
  "Start": 0,
  "End": 14,
  "TypeName": "datetimeV2.date",
  "Resolution": null
}
```

Reply:
It seems you caught a bug. Can you open an issue for this for deeper analysis?

Reply:
Tracked in #2246